src/main/java/com/puppycrawl/tools/checkstyle/site/XdocsTemplateSink.java

```diff
@@ -112,6 +113,12 @@ public void tableRows(int[] justification, boolean grid) {
      */
     private static final class CustomPrintWriter extends PrintWriter {
 
+        /** A Regex pattern to represent all kinds of newline character. */
+        private static final Pattern LINE_BREAK_ESCAPE = Pattern.compile("\\R");
+
+        /** Unix-Style newline character. */
+        private static final String NEWLINE_CHARACTER = "\n";
```

Comment:
After some clarity in other PR, let's name this as just `NEWLINE`, since we have a string here and not a character, please also adjust comment to match.

Reply:
done

Reply:
@kalpadiptyaroy always allow reviewers to resolve their own items.